Gopkg.lock

```diff
@@ -260,6 +275,19 @@
   packages = ["."]
   revision = "aabc10ec26b754e797f9028f4589c5b7bd90dc20"
 
+[[projects]]
+  name = "github.com/docker/licensing"
```

Comment:
Arf, we're getting this one because `formatter` import licensing… we really need to refactor the `formatter` package